ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -2558,3 +2563,43 @@ def test_torch_instance_pow_(dtype_and_x, as_variable, native_array):
         class_name="tensor",
         method_name="pow_",
     )
+
+
+@st.composite
+def _get_dtype_and_multiplicative_matrices(draw):
+    return draw(
+        st.one_of(
+            _get_dtype_input_and_matrices(),
+            _get_dtype_and_3dbatch_matrices(),
+        )
+    )
+
+
+# matmul
+@handle_cmd_line_args
```

Comment:
we now have 4 new decorators instead of `handle_cmd_line_args` :

1. `handle_function`: used for Ivy functional API
2. `handle_method`: used for Ivy methods
3. `handle_frontend_function`: used for frontend functions
4. `handle_frontend_method`: used for frontend methods, methods tests are now skipped temporarily.

The last decorator is the one you'll need to replace `handle_cmd_line_args` with. Unfortunately, this is the only one currently not functioning correctly, but it will be rectified soon!

Reply:
@hmahmood24 perhaps you didn't see this update?

Reply:
@simonetgordon,

Apologies I wasn't able to reply earlier. I have seen the updates and I have also refactored my tests in coherence with these updates but to no avail. The `test_tensor.py` file has still so many bugs that prevents the tests from getting collected. At this point, I'm not sure what's the best course of action for me apart from just waiting this out.

Have you asked anyone from the team to look into these bugs by now?

Reply:
@hmahmood24 Hi there, yes someone is trying to tackle these but there should have already been some fixes pushed. 

Also, I don't see the new decorator here in the PR version. Perhaps you forgot to push?

Reply:
@simonetgordon Hey, yes I didn't push my changes since the tests weren't collecting locally. I have made the changes and pushed them to remain coherent with all the other tests now.

Reply:
Hi @simonetgordon,

Any updates since it’s been almost 2 weeks since this PR got opened? Anything from my end to add here?